assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -74,9 +74,12 @@ export default function PropertySelect( props ) {
 			return false;
 		}
 
-		return select( MODULES_ANALYTICS_4 ).isLoadingPropertySummaries( {
-			hasModuleAccess,
-		} );
+		return (
+			select( MODULES_ANALYTICS_4 ).isLoadingPropertySummaries() ||
+			select( MODULES_ANALYTICS_4 ).isLoadingWebDataStreams( {
+				hasModuleAccess,
+			} )
```

Comment:
After creating a new property, the new property ID wasn't set in the `Select` dropdown because the loading time wasn't enough to select the new due to data synchronization/availability delay.